src/js/App/Sidenav/Navigation.test.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable quotes */
+import React from 'react';
+import ConnectedNavigation, { Navigation, dispatchToProps }  from './Navigation';
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+
+let settingsData = [{ id: 'overview', title: 'Overview', group: 'insights', active: true, subItems:
+    [{ id: 'someID', title: 'Clusters', default: true }] },
+{ id: 'rules', title: 'Rules', group: 'insights', active: false },
+{ id: 'topics', title: 'Topics', group: 'insights', active: false },
+{ title: 'Inventory', id: 'inventory', active: false },
+{ title: 'Remediations', id: 'remediations', active: false }];
+
+describe('Navigation', () => {
+    const initialProps = {
+        settings: settingsData,
+        activeApp: 'someApp',
+        activeLocation: 'openshift',
+        documentation: 'someDocs'
+    };
+    let initialState;
+    let mockStore;
+    let globalNavData = require('../../../../testdata/globalNav.json');
+
+    beforeEach(() =>{
+        mockStore = configureStore();
+        initialState = ({
+            chrome: {
+                globalNav: [globalNavData],
+                activeApp: 'someApp',
+                navHidden: false,
+                activeLocation: 'someLocation',
+                activeGroup: 'someGroup',
+                appId: 'someId'
+            }
+        });
+        const pathname = '/';
+        Object.defineProperty(window, 'location', {
+            value: {
+                pathname: pathname
+            },
+            writable: true
+        });
+    });
+    it('should render corectly', () =>{
+        let props = {
+            ...initialProps,
+            appId: 'overview',
+            activeGroup: 'someID'
+        };
+        const mockSelect = jest.fn();
+        const mockClick = jest.fn();
+        const store = mockStore(initialState);
+        const wrapper = shallow(<Navigation onSelect= { mockSelect } onClick={ mockClick } store={ store }{ ...props }/>);
```

Comment:
Spacing is a bit off.
```suggestion
        const wrapper = shallow(<Navigation onSelect={ mockSelect } onClick={ mockClick } store={ store } { ...props }/>);
```

Reply:
done